centreon/src/Core/Resources/Infrastructure/Repository/DbReadResourceRepository.php

```diff
@@ -542,7 +542,7 @@ private function findHostsStatusCountByAccessGroupIds(array $accessGroupIds, Res
 
         $query .= ! empty($searchSubRequest) ? $searchSubRequest . ' AND ' : ' WHERE ';
         $query .= <<<'SQL'
-            resources.type=1 AND resources.name NOT LIKE "_Module_%"
+            resources.enabled = 1 AND resources.type=1 AND resources.name NOT LIKE "_Module_%"
```

Comment:
When using the LIKE expression, the underscore is a wildcard for any character.
You must therefore escape it if you want to use it as an underscore.
```suggestion
            resources.enabled = 1 AND resources.type=1 AND resources.name NOT LIKE "\_Module\_%"
```